distributed/config.py

```diff
@@ -23,6 +23,7 @@
     "bandwidth": "distributed.scheduler.bandwidth",
     "default-data-size": "distributed.scheduler.default-data-size",
     "transition-log-length": "distributed.scheduler.transition-log-length",
+    "events-log-length": "distributed.scheduler.events-log-length",
```

Comment:
These aliases are for backwards compatibility.  You don't need to add this here (no one was using the "events-log-length" config value previously.

Reply:
good to know, will remove it, thanks!